From 3119a0b7c8b479faec78a8a46644f8b509fc5f86 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 12 Jul 2017 16:11:28 -0700
Subject: [PATCH] TOOLS-1807 node updates for july 2017 node.js security
 releases Reviewed by: Todd Whiteman <todd.whiteman@joyent.com> Approved by:
 Todd Whiteman <todd.whiteman@joyent.com>

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index fa2956c..b901ca5 100644
--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@ SMF_MANIFESTS = $(shell ls smf/manifests/*.xml)
 NODEUNIT	:= ./node_modules/.bin/nodeunit
 CLEAN_FILES += ./node_modules
 
-NODE_PREBUILT_VERSION=v4.6.1
+NODE_PREBUILT_VERSION=v4.8.4
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
 	# Allow building on other than image sdc-minimal-multiarch-lts@15.4.1.
-- 
2.21.0

